gui/layers/_markers_layer.py

```diff
@@ -341,9 +334,108 @@ def _set_view_slice(self, indices):
             sizes = 0
 
         self._node.set_data(
-            data[::-1], size=sizes, edge_width=self.edge_width, symbol=self.symbol,
-            edge_width_rel=self.edge_width_rel,
+            data[::-1], size=sizes, edge_width=self.edge_width,
+            symbol=self.symbol, edge_width_rel=self.edge_width_rel,
             edge_color=self.edge_color, face_color=self.face_color,
             scaling=self.scaling)
         self._need_visual_update = True
         self._update()
+
+    def _get_coord(self, position, indices):
+        max_shape = self.viewer.dimensions.max_shape
+        transform = self._node.canvas.scene.node_transform(self._node)
+        pos = transform.map(position)
+        pos = [clip(pos[1], 0, max_shape[0]-1), clip(pos[0], 0,
+                                                     max_shape[1]-1)]
+        coord = copy(indices)
+        coord[0] = int(pos[1])
+        coord[1] = int(pos[0])
```

Comment:
This is seriously killing me, but I'll trust that it'll be fixed in the future...

Reply:
you'll have to trust us ;-) next phase of the refactor - to fix the dimensions indexing - once we agree on the basic file structure and UI layout